essence-of-live-coding/src/LiveCoding/Cell/Util.hs

```diff
@@ -68,6 +68,14 @@ keepJust = feedback Nothing $ arr keep
     keep (_, Just a) = (Just a, Just a)
     keep (Just a, Nothing) = (Just a, Just a)
 
+-- | Hold the first value and output it indefinitely.
+hold :: (Data a, Monad m) => Cell m a a
```

Comment:
Yampa calls the function `keep` above [`hold`](https://hackage.haskell.org/package/Yampa-0.13.1/docs/FRP-Yampa.html#v:hold). Perhaps better to maintain the same name ?